src/equations/compressible_euler_1d.jl

```diff
@@ -1,10 +1,24 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
 @doc raw"""
-    CompressibleEulerEquations1D
+    CompressibleEulerEquations1D(γ)
```

Comment:
I thought this would be a case for
> Docstrings and comments can and should use Unicode characters where it helps understanding.

Reply:
Right, that's why I am not sure if I made the right call. It's a corner case, since in this case it does not refer to a variable in a function, but to an actually existent argument name. To me it feels very weird to document a parameter `γ` while in fact it is called `gamma`, and in my case the Greek letter does not improve legibility or understandability (especially since it is, depending on the used font, not very hard to be mistaken for a `y`).

Anyhow, I'll leave it up to you!